catalog/Dockerfile

```
@@ -39,7 +39,9 @@ ENV AIRFLOW__LOGGING__REMOTE_BASE_LOG_FOLDER=s3://openverse-airflow-logs
 
 
 USER root
-RUN apt-get update \
+# FIXME: Remove this exclusion once the mysql PGP key is no longer expired
+RUN rm /etc/apt/sources.list.d/mysql.list \
```

Since we don't use anything from these repos, [we can safely drop all of them](https://askubuntu.com/a/944924).
```suggestion
RUN rm /etc/apt/sources.list.d/* \
```

---

This list includes some sources that we _do_ need, unfortunately. I'd rather take a targeted approach in this case!